spec/support/feature_helpers/publish_pages.rb

```diff
@@ -51,5 +51,61 @@ def publish_provider_courses_show_page
     def publish_provider_courses_details_page
       @publish_provider_courses_details_page ||= PageObjects::Publish::ProviderCoursesDetails.new
     end
+
+    def new_level_page
+      @new_level_page ||= PageObjects::PublishInterface::Courses::NewLevel.new
```

Comment:
These should just be `Publish` not `PublishInterface` as we got rid of it

Reply:
Fixed in https://github.com/DFE-Digital/teacher-training-api/pull/2471/commits/dbeee9f936b08132662a4858574470e7d9a9acbc